Library/Homebrew/dev-cmd/man.rb

```diff
@@ -70,6 +70,9 @@ def build_man_page
     variables[:maintainers] = readme.read[/(Linuxbrew's other current maintainers are .*\.)/, 1]
                                     .gsub(/\[([^\]]+)\]\([^)]+\)/, '\1')
 
+	variables[:former_maintainers] = readme.read[/(Former maintainers .*\.)/, 1]
+                                           .gsub(/\[([^\]]+)\]\([^)]+\)/, '\1')
```

Comment:
Minor style issues caused I think by using a tab rather than spaces.

```
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/dev-cmd/man.rb:73:1: C: Tab detected.
	variables[:former_maintainers] = readme.read[/(Former maintainers .*\.)/, 1]
^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/dev-cmd/man.rb:73:2: C: Inconsistent indentation detected.
	variables[:former_maintainers] = readme.read[/(Former maintainers .*\.)/, 1] ...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/dev-cmd/man.rb:74:44: C: Align .gsub with .read on line 73.
                                           .gsub(/\[([^\]]+)\]\([^)]+\)/, '\1')
                                           ^^^^^
```